index.js

```diff
@@ -2743,24 +2743,28 @@
         )(s);
       });
 
-  //# parseJson :: String -> Maybe Any
+  //# parseJson :: TypeRep a -> String -> Maybe a
   //.
-  //. Takes a string which may or may not be valid JSON, and returns Just
-  //. the result of applying `JSON.parse` to the string if valid; Nothing
-  //. otherwise.
+  //. Takes a [type representative](#type-representatives) and a string which
+  //. may or may not be valid JSON, and returns Just the result of applying
+  //. `JSON.parse` to the string *if* the result is of the specified type
```

Comment:
It works for me, but the one thing that's not clear from this description is the handling of invalid strings.

What do you think of this phrasing?

_… and returns Just `x` if the given string is a JSON representation of `x`, a value of the specified type (according to [`is`][1]); Nothing otherwise._


[1]: http://sanctuary.js.org/#is